runtime/runtime_memory_metering_test.go

```diff
@@ -1071,3 +1071,150 @@ func TestMemoryMeteringErrors(t *testing.T) {
 		assert.ErrorIs(t, err, testMemoryError{})
 	})
 }
+
+func TestMeterEncoding(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("string", func(t *testing.T) {
+
+		t.Parallel()
+
+		rt := newTestInterpreterRuntime()
+		rt.defaultConfig.AtreeValidationEnabled = false
+
+		address := common.MustBytesToAddress([]byte{0x1})
+		storage := newTestLedger(nil, nil)
+		meter := newTestMemoryGauge()
+
+		runtimeInterface := &testRuntimeInterface{
+			storage: storage,
+			getSigningAccounts: func() ([]Address, error) {
+				return []Address{address}, nil
+			},
+			log:         func(message string) {},
+			meterMemory: meter.MeterMemory,
+		}
+
+		text := "A quick brown fox jumps over the lazy dog"
+
+		err := rt.ExecuteTransaction(
+			Script{
+				Source: []byte(fmt.Sprintf(`
+                transaction() {
+                    prepare(acc: AuthAccount) {
+                        var s = "%s"
+                        acc.save(s, to:StoragePath(identifier: "some_path")!)
```

Comment:
Maybe just use a path literal:

```suggestion
                        acc.save(s, to: /storage/some_path)
```